app/assets/stylesheets/semantic-ui/elements/_button.scss

```diff
@@ -196,9 +184,9 @@
 .ui.disabled.button:hover,
 .ui.disabled.active.button {
   cursor: default;
-  opacity: 0.45 !important;
+  opacity: 0.6 !important;
   background-image: none !important;
-  box-shadow: none !important;
+  box-shadow: 0 3px 6px rgba(0, 0, 0, 0.15);
```

Comment:
This is a box-shadow definition being used for multiple button styles, replace it with a variable